metaflow/main_cli.py

```diff
@@ -437,8 +437,16 @@ def import_from(profile, input_filename):
     help="Configure a named profile. Activate the profile by setting "
     "`METAFLOW_PROFILE` environment variable.",
 )
-def sandbox(profile):
-    overwrite_config(profile)
+@click.option(
+    "--overwrite/--no-overwrite",
+    "-o/",
+    default=False,
+    show_default=True,
+    help="Overwrite profile configuration without asking",
+)
+def sandbox(profile, overwrite):
+    if not overwrite:
```

Comment:
Shouldn't this be the other way round?

Reply:
No, its just `overwrite_confg()` is a little confusingly named. It should really be `ask_to_overwrite_config()`. The idea is that if `--overwrite` is set, it doesn't get called, and the config is overwritten without asking.

Maybe the flag should be `--quiet-overwrite/--no-quiet-overwrite` 🤔

Reply:
can we just change `overwrite_confg` to `ask_to_overwrite_config` and ship this PR?